CHANGELOG.md

```diff
@@ -8,6 +8,7 @@ All notable changes to this project will be documented in this file.
 
 ### Fixed
 - Text input box theming
+- Password repository held in non-hidden storage often fails
```

Comment:
Would read better to me if it were formulated positively (e.g. "no longer fails").

Reply:
Done